.github/scripts/parse_pr_standards.py

```
@@ -0,0 +1,119 @@
+        suffix=".txt",
+        delete=False,
+        dir="/tmp",
+    ) as tmp:
+        tmp.write(body)
+        body_file = tmp.name
+
+    try:
+        validator = Path(
+            ".claude/skills/github/scripts/pr/validate_pr_description.py"
+        )
+        result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)